<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keygenClass</class>
 <widget class="QWidget" name="keygenClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>keygen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="code_label">
     <property name="text">
      <string>Кодовое число</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QLineEdit" name="code_edit"/>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLabel" name="id_label">
     <property name="text">
      <string>Id устройства</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QListView" name="dev_list"/>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="add_dev_btn">
     <property name="text">
      <string>добавить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="get_id_btn">
     <property name="text">
      <string>получить id</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="save_button">
     <property name="text">
      <string>создать ключ</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLineEdit" name="dev_edit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
